torch/_tensor.py

```diff
@@ -55,9 +55,6 @@ def _rebuild_from_type(func, type, args, dict):
 
 
 def _rebuild_from_type_v2(func, new_type, args, state):
-    if new_type is Tensor:
```

Comment:
This is BC-breaking right?
Not that we don't want this, but we should document it as such in the release notes.